pkg/claimsstore/signedclaimpersister.go

```diff
@@ -115,9 +122,30 @@ func (p *SignedClaimPGPersister) AddCredential(claim *claimtypes.ContentCredenti
 }
 
 // GetCredentialByHash returns a credential from a hash taken from the associated merkle tree claim
-func (p *SignedClaimPGPersister) GetCredentialByHash(hash string) (*claimtypes.ContentCredential, error) {
+func (p *SignedClaimPGPersister) GetCredentialByHash(hash string) (*claimtypes.ContentCredential,
+	error) {
+	signedClaim := &SignedClaimPostgres{}
+	bytes, err := hex.DecodeString(hash)
+	if err != nil {
+		return nil, err
+	}
+
+	mHash, err := multihash.EncodeName(bytes, "keccak-256")
+	if err != nil {
+		return nil, err
+	}
+	mHashString := hex.EncodeToString(mHash)
+	if err := p.db.Where(&SignedClaimPostgres{Hash: mHashString}).First(signedClaim).Error; err != nil {
```

Comment:
Up to you, but could probably call `GetCredentialByMultihash` here in case that query changes later.